FlashFeetAPI/src/main/java/org/example/cli/JobRole.java

```diff
@@ -0,0 +1,27 @@
+package org.example.cli;
+
+public class JobRole {
+
+    public JobRole(){
+
+    }
+
+    private String jobRole;
```

Comment:
When we create a job role, should we not use the constructor to pass in the parameters and set them?